rep-2000.rst

```diff
@@ -1179,6 +1179,120 @@ Package manager use for dependencies:
++==========================+========================+===============+============================+================================+
+| rmw_fastrtps_cpp         | eProsima Fast-DDS      | Tier 1        | All Platforms              | All Architectures              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| rmw_cyclonedds_cpp       | Eclipse Cyclone DDS    | Tier 1        | All Platforms              | All Architectures              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| rmw_connextdds           | RTI Connext            | Tier 1        | Ubuntu, Windows, and macOS | All Architectures except arm64 |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-DDS      | Tier 2        | All Platforms              | All Architectures              |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+| rmw_gurumdds_cpp         | GurumNetworks GurumDDS | Tier 3        | Ubuntu and Windows         | All Architectures except arm32 |
++--------------------------+------------------------+---------------+----------------------------+--------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minimum language requirements:
+
+- C++17
+- Python 3.8
+
+
+Dependency Requirements:
+
++----------------+---------------------------------+-------------------------------------------------------------------+
+|                |        Required Support         |                        Recommended Support                        |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Package        |  Ubuntu Noble  |  Windows 10**  |  RHEL 9  |    macOS**    | Debian Bookworm  |   OpenEmbedded**    |
++================+================+================+==========+===============+==================+=====================+
+| CMake          |     3.28.3     |     3.29.2     |  3.26.5  |    3.31.1     |      3.25.1      |       3.22.3        |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| EmPY           |     3.3.4      |     3.3.4      |  3.3.4a  |                         3.3.4                          |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Gazebo         |     Ionic*     |      N/A       |   N/A    |    Ionic*     |      Ionic*      |         N/A         |
```

Comment:
@azeey to confirm - targeting Gazebo Ionic for Kilted Kaiju on Ubuntu Noble.

Reply:
That is correct.